src/services/billing/cost-management/cost-analysis/modules/CostAnalysisQueryFilter.vue

```diff
@@ -134,65 +151,97 @@ export default {
         const handleClickRefresh = async () => {
             // todo
         };
+        const handleClickSetFilter = () => {
+            state.setQueryModalVisible = true;
+        };
+
+        const dateFormatter = date => dayjs(date).format('YYYY/MM/DD');
```

Comment:
```suggestion
        const dateFormatter = date => dayjs.utc(date).format('YYYY/MM/DD');
```

이렇게 해주세요! 여기선 무조건 UTC를 사용하는데, 이렇게 그냥 dayjs로 묶으면 자동으로 브라우저 시간을 따르거든용